src/Ledger/Utxo.lagda

```diff
@@ -113,6 +113,68 @@ The UTxO transition system is given in Figure~\ref{fig:rules:utxo-shelley}.
 \end{itemize}
 \end{NoConway}
 
+The deposits have been reworked since the original Shelley design. We
+now track the amount of every deposit individually. This fixes an
+issue in the original design: An increase in deposit amounts would
+allow an attacker to make lots of deposits before that change and
+refund them after the change. The additional funds necessary would
+have been provided by the treasury. Since changes to protocol
+parameters were (and still are) known publicly and guaranteed before
+they are enacted, this comes at zero risk for an attacker. This means
+the deposit amounts could realistically never be increased. This issue
+is gone with the new design.
+
+Similar to \ScriptPurpose, \DepositPurpose carries the information
+what the deposit is being made for. The deposits are stored in the
```

Comment:
```suggestion
Similar to \ScriptPurpose, \DepositPurpose specifies the purpose of the deposit. The deposits are stored in the
```

Reply:
This is intentional. The sentence 'DepositPurpose specifies the purpose of the deposit' is tautological. The purpose of this sentence is to give an alternative explanation. Maybe the explanation I have is still not that great, but we'll need to have at least one full read through later anyway, so we can fix it then.